api/build.go

```diff
@@ -990,6 +990,14 @@ func RestartBuild(c *gin.Context) {
 	b.SetRuntime("")
 	b.SetDistribution("")
 
+	// update the PR event if action was never set
```

Comment:
```suggestion
	// update the PR event action if action was never set
	// for backwards compatibility with pre-0.14 releases
```
a little nitpicky, but thought we can be more explicit. thoughts?